Cargo.toml

```diff
@@ -21,25 +21,27 @@ ethereum-types = "0.9.0"
 futures = "0.3.5"
 futures-timer = "3.0.2"
 hex = "0.4"
-libp2p = { version = "0.21", default-features = false, features = ["tcp-tokio", "secio", "yamux", "mplex", "dns", "gossipsub", "request-response"] }
+libp2p = { version = "0.24", default-features = false, features = ["tcp-tokio", "secio", "yamux", "mplex", "dns"] }
 log = "0.4"
 num = "0.3"
 num256 = "0.2"
 pem = "0.8"
-reqwest = { version = "0.10", default-features = false, features = ["json", "native-tls"] }
+reqwest = { version = "0.10.8", default-features = false, features = ["json", "native-tls"] }
```

Comment:
Can you explain this change? the lock file should sort that out for you.

Reply:
I was getting dependency conflicts, I will try again